src/main/webapp/style.css

```diff
@@ -0,0 +1,75 @@
+.container {
+    font-family: "Montserrat";
+    display: flex;
+    flex-direction: column;
+}
+
+.home-title {
+    color: rgb(16, 89, 16);
+    font-size: 5em;
+    margin: 2em 0 0;
+    text-align: center;
+}
+
+.home-subtitle {
+    color: rgb(19, 73, 19);
```

Comment:
If you're planning to use the same color it would be helpful to have a variable named textColor, primaryColor, or the actual color.